### PR TITLE
feat: populate sandbox and discontinue app

### PR DESCRIPTION
**Pull request by rossbulat**
Populates the sandbox with simple components and discontinues `app`.

### CODE REVIEW

File: sandbox/src/Theme.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import { Router } from "./Router";
 import { useTheme } from "./contexts/Theme";
 
-// App-wide theme classes are inserted here.
+// App-wide theme classes are inserXted here.
```
Comment (TingALin) [inline sandbox/src/Theme.tsx:7]:
```suggestion
// App-wide theme classes are inserted here.
```

**Review by TingALin — DISMISSED** [review]
A note to myself: The APP is being continued on the front page repo because we don't want to commit history to mess up the library committ history, and the APP has been replaced with sandbox it like an easier way to test the components.

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build:directory": "node builder/run.mjs -t directory:build",
     "build:mock": "npm run build:mock --workspaces --if-present",
-    "clear": "npm run --workspaces clear && rm -rf node_modules coverage",
+    "clear": "npm run --workspaces clear && rm -rf node_modules yarn.lock",
```
Comment (wirednkod) [inline package.json:14]: You should not "clean" yarn.lock this way - we should keep this always

File: sandbox/src/components/Header/index.tsx [hunk 3]
```diff
@@ -2,42 +2,40 @@
...
-      amplitudeX: 0.2,
-      amplitudeY: 0.2,
-    },
-  });
+  const { mode, toggleMode, theme, setTheme } = useTheme();
+  const allThemes = [
+    ["Polkadot", "polkadot-relay"],
+    ["Kusama", "kusama-relay"],
+    ["Westend", "westend-relay"],
+    ["XCM", "xcm"],
```
Comment (wirednkod) [inline sandbox/src/components/Header/index.tsx:14]: lets remove XCM theme... its Kusama anyways

File: sandbox/src/components/Page/index.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+SPDX-License-Identifier: GPL-3.0-only */
+
+import { useTheme } from "../../contexts/Theme";
+// Import the base cloud-core styles to ensure the theme styles are applied instead of doc styles.
+import "../../../../packages/cloud-core/dist/css/styles/index.css";
+// Import each supported theme here.
+import "../../../../packages/cloud-core/dist/accent/polkadot-relay.css";
+import "../../../../packages/cloud-core/dist/accent/kusama-relay.css";
+import "../../../../packages/cloud-core/dist/accent/westend-relay.css";
+import "../../../../packages/cloud-core/dist/accent/xcm.css";
```
Comment (wirednkod) [inline sandbox/src/components/Page/index.tsx:11]: same for xcm

**Review by wirednkod — COMMENTED** [review]
Some nips and tucks but this is very nicely done

File: sandbox/src/components/Header/index.tsx (hunk 3 above)
Comment (rossbulat) [reply sandbox/src/components/Header/index.tsx:14]: Yessir

**Review by wirednkod — APPROVED** [review]

**Pull request merged**